samples/node/fleet_provisioning/README.md

```diff
@@ -79,6 +79,13 @@ npm install
 node ./index.js --endpoint <endpoint> --cert <file> --key <file> --template_name <template name> --template_parameters <template parameters>
 ```
 
+You can also run the sample with `--mqtt5` to run it with Mqtt5 Client
```

Comment:
Trivial: "You may run the sample using an Mqtt5 Client by adding the `--mqtt5` argument."

Does the argument also have to be set to "true" or can it be blank? Asking as the json above sets it to "true"